bootstrap.tf

```diff
@@ -9,7 +9,7 @@ resource "cloudscale_server" "bootstrap" {
     type = "private"
     addresses {
       address     = cidrhost(var.privnet_cidr, 10)
```

Comment:
Currently, this will generate a bogus address when an external subnet is specified, unless that subnet is configured with CIDR 172.18.200.0/24.

Ideally, we could use a Terraform data source to extract the supplied private network's CIDR, but the cloudscale.ch Terraform provider doesn't support data sources (cf. https://github.com/cloudscale-ch/terraform-provider-cloudscale/issues/40).

Therefore, users will need to ensure that they provide the correct `privnet_cidr` even when they supply an existing private network, see my suggestion in `variables.tf`.